planet/cli/data.py

```diff
@@ -480,8 +495,133 @@ async def search_update(ctx,
         echo_json(items, pretty)
 
 
+@data.command()
+@click.pass_context
+@translate_exceptions
+@coro
+@click.argument("item_type", type=str, callback=check_item_type)
+@click.argument("item_id")
+@click.argument("asset_type_id")
```

Comment:
OK this is supa confusing but if you look at https://developers.planet.com/docs/apis/data/reference/#tag/Items-and-Assets/operation/ListItemAssets then the argument here (and everywhere else in these CLI commands) should be named `asset_type`

Sorry I missed this last time.

Reply:
I've been writing it as `asset_type_id` because that's what we use for those functions in the client, but `asset_type` makes way more sense, I'm on it.